e2e-tests/development-runtime/src/components/class-component.js

```diff
@@ -1,8 +1,11 @@
 import React, { Component } from "react"
 
 class ClassComponent extends Component {
-  state = {
-    custom: true,
+  constructor(props) {
+    super(props)
+    this.state = {
+      custom: true,
+    }
```

Comment:
Can we add a eslint disable instead? This syntax is handled by babel and we had an issue a few weeks ago where babel broke something and we could see it in this test.

Reply:
Sure. I am also confused about why I can't reproduce htis locally, but it's happening on a lot of unrelated PRs in CI (including in the 2.32 branch)